boot/zephyr/sample.yaml

```diff
@@ -5,9 +5,10 @@ sample:
 tests:
   sample.bootloader.mcuboot:
     tags: bootloader_mcuboot
-    platform_allow: nrf52840dk_nrf52840 frdm_k64f
```

Comment:
I have to consult this line. I'll response soon.

Reply:
Twister will run this testcase for all, listed bellow integration platform (if run with `--integration`). This change causes that it will be possible to runt this testcase for any platform (specified in commndaline).